script/tool/lib/src/create_all_plugins_app_command.dart

```diff
@@ -259,4 +268,75 @@ dev_dependencies:${_pubspecMapString(pubspec.devDependencies)}
 
     return buffer.toString();
   }
+
+  Future<int> _genNativeBuildFiles() async {
+    // Only run on macOS.
+    // Other platforms don't need generation of additional files.
+    if (!io.Platform.isMacOS) {
```

Comment:
We may need this for other platforms in the future, and `pub get` will be part of building the app anyway, so we can just do this unconditionally and keep the codepaths the same on all platforms.